packages/kit/src/runtime/server/page/render.js

```diff
@@ -151,7 +152,11 @@ export async function render_response({
 			const fetch = globalThis.fetch;
 			let warned = false;
 			globalThis.fetch = (info, init) => {
-				if (typeof info === 'string' && !/^\w+:\/\//.test(info)) {
+				// The regex below does not strictly check if a URL is absolute, but checks for the existance
+				// of a scheme, which implies that it is absolute when the scheme is e.g., http or https.
+				// And it will accept data URLs, which don't require a network request.
+				// Regex based on scheme definition at https://www.rfc-editor.org/rfc/rfc3986#section-3.1
```

Comment:
this seems pretty self-explanatory from the nice variable name we have now and the link is already included in that variable's description
```suggestion
```